presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/Predicates.java

```diff
@@ -15,26 +15,18 @@
 
 import java.util.function.Predicate;
 
+import static com.google.common.base.Predicates.alwaysFalse;
+
 public class Predicates
 {
     private Predicates() {}
 
     public static <T> Predicate<T> isInstanceOfAny(Class... classes)
```

Comment:
Not really, because `Predicates::instanceOf` takes only one class and we need two classes. Using, `or(instanceOf, isntanceOf)` won't be readable